lib/compat/wordpress-6.7/class-gutenberg-rest-posts-controller-6-7.php

```diff
@@ -485,7 +485,6 @@ public function get_collection_params() {
 
 		if ( post_type_supports( $this->post_type, 'post-formats' ) ) {
 			$query_params['format'] = array(
-				'default'     => 'standard',
```

Comment:
I think when no formats are set, all formats should be displayed. Removing the default argument here solves that.